coffee/controller/list-item-ctrl.coffee

```diff
@@ -1,7 +1,37 @@
 ListItemCtrl = ($scope, githubService) ->
 
   $scope.getNotifications = () ->
-    githubService.notifications($scope.repo)
+    answer = githubService.notifications($scope.repo)
+    if !answer.value
+      "empty"
+    else answer
+
+  $scope.dateUpdated = () -> 
+    date = $scope.repo.updated_at
+    currentDate = new Date(date)
+    day = currentDate.getDate()
+    month = currentDate.toLocaleString('en-us', { month: "short" })
+    year = currentDate.getFullYear()
+    updateOn = "Updated on " + day + " " + month + " " + " " +year
+
+
+  
+
+  getColor = () -> 
+    lengthWord = $scope.repo.language.length
+    colorNumber = lengthWord % 7
+    switch colorNumber
+      when 0 then "red"
+      when 1 then "orange"
+      when 2 then "yellow"
+      when 3 then "green"
+      when 4 then "blue"
+      when 5 then "violet"
+      else "black"
+
+  $scope.myObj = {
```

Comment:
maybe can add this `color` information into `$scope.repo` instead of adding a new variable object here?